Atomic/syscontainers.py

```diff
@@ -7,7 +7,7 @@
 from string import Template
 import calendar
 import shutil
-import stat # pylint: disable=bad-python3-import
+# pylint: disable=bad-python3-import
```

Comment:
I think this can be dropped.

Reply:
Thanks for the review! and ah, yea.. i was thinking about removing it earlier, but then. there are pylint errors after I remove those. Hmm, weirdo, the pylint error no longer showed up this time, #facepalm. Will drop it then :P.